SECURITY.MD

```diff
@@ -8,7 +8,11 @@ If you believe you have found a security vulnerability in any Microsoft-owned re
 
 ## Reporting Security Issues
 
-**Please do not report security vulnerabilities through public GitHub issues.** Instead, please report them to the Microsoft Security Response Center at [secure@microsoft.com](mailto:secure@microsoft.com).  If possible, encrypt your message with our PGP key; please download it from the [Microsoft Security Response Center PGP Key page](https://technet.microsoft.com/en-us/security/dn606155).
+**Please do not report security vulnerabilities through public GitHub issues.**
+
+Instead, please report them to the Microsoft Security Response Center (MSRC) at https://msrc.microsoft.com/create-report.
```

Comment:
I think GFM automatically makes that link text a clickable link, but would you mind making that explicit?

Also, the link targets (msrc.microsoft.com and msrc.microsoft.com/create-report) are both very broken looking to me on Edge and Chrome. (Not the link -- the actual pages.) -- Seeing 500s coming back on some resources and the page looks like this:

![image](https://user-images.githubusercontent.com/732166/64230515-e2b3c700-cea1-11e9-8e94-09aeae574537.png)

Reply:
Made link explicit and sent mail on 500s. I was not having any issues with the portal prior to sending this PR, or now, but hopefully the portals team can track it down.